about-face.js

```diff
@@ -73,3 +75,63 @@ Hooks.on("renderSceneConfig", renderSceneConfigHandler);
 Hooks.on("renderSceneConfig", asyncRenderSceneConfigHandler);
 Hooks.on("renderTokenConfig", renderTokenConfigHandler);
 Hooks.on("renderSettingsConfig", renderSettingsConfigHandler);
+
+
+
+// Main function which gets the users cursor coordinates and updates the selected tokens "direction" arrow points it at those coordinates.
+function targetAndRotateToCursor() {
+	// test logs for PIXI stuff
+    // console.log(canvas);
+    // console.log(canvas.app);
+    // console.log(canvas.app.renderer);
+    // console.log(canvas.app.renderer.plugins);
+    // console.log(canvas.app.renderer.plugins.interaction);
+    // console.log(canvas.app.renderer.plugins.interaction.mouse);
+
+	const globalMousePos = new PIXI.Point(canvas.app.renderer.plugins.interaction.rootPointerEvent.client.x, canvas.app.renderer.plugins.interaction.rootPointerEvent.client.y);    const localMousePos = canvas.app.stage.toLocal(globalMousePos);
+
+    // Iterate over all controlled tokens, this is useful for changing a group direction.
+    for (const sourceToken of canvas.tokens.controlled) {
+        if (!sourceToken) continue;
+
+        const sourceX = sourceToken.x + sourceToken.width / 2;
+        const sourceY = sourceToken.y + sourceToken.height / 2;
+
+        // Calculate the angle between the source token and the mouse cursor
+        const angleDeg = calculateAngle(sourceX, sourceY, localMousePos.x, localMousePos.y);
+        
+        // Update the token's direction
+        sourceToken.document.update({
+            "flags.about-face.direction": angleDeg
+        });
+    }
+}
+
+// 'T' key event listener, this listens for when the T key is pressed and turns the token that direction, also works wih groups.
+document.addEventListener("keydown", (event) => {
+    if (event.code === "KeyT" && !event.altKey && !event.ctrlKey && !event.shiftKey && !event.metaKey) {
+        targetAndRotateToCursor();
+    }
+});
```

Comment:
You should've implemented it as a Foundry keybind on the init hook instead of using pure JS.

Also, T is a default Foundry keybind, so it should be left up to the user's choice. I'm not sure how many users _want_ that feature and would get surprised by that behavior, so it shouldn't be available without them opting in. And having to tell people "unbind your T to use this module" is poor UX.